staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -181,6 +181,10 @@ type VolumeSource struct {
 	//
 	// +optional
 	Ephemeral *EphemeralVolumeSource `json:"ephemeral,omitempty" protobuf:"bytes,29,opt,name=ephemeral"`
+	// oci represents a OCI object (image or artifact) pulled and mounted on the kubelet's host machine.
+	// +featureGate=OCIVolume
+	// +optional
+	OCI *OCIVolumeSource `json:"oci,omitempty" protobuf:"bytes,30,opt,name=oci"`
```

Comment:
I am torn on whether to argue that this should be called `Image` - historically that would be correct, since Kubernetes does not require the CRI to implement OCI images (you could be based on docker v2 spec, or implement a transformation layer).

Reply:
My personal opinion is that `oci` fits greatly for both, images and artifacts. Maybe also `ociObject` if it's not specific enough.

cc @sallyom @mrunalp @haircommander

Reply:
I had similar feelings when I saw this name first compared to existing `Image`. I think naming it `Image` will make it more future proof if we want to add support for some other tranport later on.

Reply:
Makes sense, changed the image name to `Image`.